CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased: mitmproxy next
 
+* **Multiple Browsers:** The `browser.start` command may be executed more than once to start additional
+  browser sessions.
+
```

Comment:
Can you please move this to the full changelog below? I know it's a bit confusing with the big "New Proxy Core" block. :)